### PR TITLE
Complete AttributeFilter integration with AttributeFilterHandler

### PR DESCRIPTION
**Pull request by kandl**
- Add more AttributeFilter customization possibilites (status bar, search bar)
- Improve naming
- Simplify typings
- Add getLimitingAttributeFiltersAttributes() to AttributeFilterHandler (to display "is filtered by:")
- Align styles with UX designs
- Minor fixes and code improvements
- Fix storybooks

JIRA: RAIL-4408

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-filters/src/AttributeFilter@next/AttributeFilterButton.tsx [hunk 1]
```diff
@@ -16,7 +17,8 @@ export const AttributeFilterButton: React.FC<IAttributeFilterButtonProps> = (pro
         <AttributeFilterBase
             {...props}
             DropdownButtonComponent={props.DropdownButtonComponent ?? AttributeFilterDropdownButton}
-            LoadingComponent={props.LoadingComponent ?? LoadingComponent}
+            LoadingComponent={props.FilterLoading ?? props.LoadingComponent ?? LoadingComponent}
+            ErrorComponent={props.FilterError}
```
Comment (ivanmjartan) [inline libs/sdk-ui-filters/src/AttributeFilter@next/AttributeFilterButton.tsx:21]: Maybe we should check props for FilterError and if it is not defined than set default one ?

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9327/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11161/ : FAILURE in 9m 30s

File: libs/sdk-ui-filters/src/AttributeFilter@next/Components/ElementsSelect/AttributeFilterElementsSelect.tsx [hunk 2]
```diff
@@ -33,48 +60,92 @@ export const AttributeFilterElementsSelect: React.FC<IAttributeFilterElementsSel
...
-            isLoadingNextPage={isLoadingNextPage}
-            nextPageItemPlaceholdersCount={nextPageSize}
-            onLoadNextPage={onLoadNextPage}
-        />
+        <>
+            <InvertableSelect<IAttributeElement>
+                className="gd-attribute-filter-list"
+                width={INTERNAL_LIST_WIDTH}
+                height={
+                    isLoading || (!isLoading && items.length === 0)
```
Comment (ivanmjartan) [inline libs/sdk-ui-filters/src/AttributeFilter@next/Components/ElementsSelect/AttributeFilterElementsSelect.tsx:90]: extract this height calculation into method and memo this.

**Review by ivanmjartan — APPROVED** [review]

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9328/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11162/ : FAILURE in 10m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9332/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11164/ : FAILURE in 4m 08s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9333/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11165/ : SUCCESS in 16m 40s

**Review by ivanmjartan — APPROVED** [review]

**Review by ivanmjartan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2865

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2638/ : SUCCESS in 3m 40s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2939/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1772/ : SUCCESS in 15m 45s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11166/ : SUCCESS in 16m 44s

**Pull request merged**